packages/openapi-generator/src/file-serializer/schema.spec.ts

```diff
@@ -18,7 +18,8 @@ describe('serializeSchema for object schemas', () => {
           {
             name: 'simpleProperty',
             required: true,
-            schema: { type: 'integer' }
+            schema: { type: 'integer' },
```

Comment:
can you perhaps add a small unit test here that a schema with this properties is correctly parsed.